chapters/classes.tex

```diff
@@ -1028,12 +1044,13 @@ \section{Balanced Models}\doublelabel{balanced-models}
 
 A model or block class is \firstuse{locally balanced} if the
 \emph{local number of unknowns} is identical to the \emph{local equation size} for all legal
-values of constants and parameters {[}\emph{respecting final bindings
-and min/max-restrictions. A tool shall verify the \emph{locally balanced}
-property for the actual values of parameters and constants in the
-simulation model. It is a quality of implementation for a tool to verify
-this property in general, due to arrays of (locally) undefined sizes,
-conditional declarations, for loops etc.}{]}\emph{.}
+values of constants and parameters.
+
+\begin{nonnormative}
+Here, \emph{leval values} must respect final bindings and min/max-restrictions. A tool shall verify the \emph{locally balanced} property for the actual
```

Comment:
Either the font is broken or it should be changed from "leval" to "legal".

Reply:
Thanks!  Fixed.